instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/example/go.mod

```diff
@@ -0,0 +1,21 @@
+module go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/example
+
+go 1.14
+
+replace (
+	go.opentelemetry.io/contrib => ../../../../../..
```

Comment:
since this is an example I think we should just use the released version right? If users want it they can always point this to local project using `replace`

Reply:
Example can reflect the latest change in local by replacing path